hudi-utilities/src/main/java/org/apache/hudi/utilities/deltastreamer/HoodieDeltaStreamer.java

```diff
@@ -396,10 +424,10 @@ public int hashCode() {
               baseFileFormat, propsFilePath, configs, sourceClassName,
               sourceOrderingField, payloadClassName, schemaProviderClassName,
               transformerClassNames, sourceLimit, operation, filterDupes,
-              enableHiveSync, maxPendingCompactions, continuousMode,
+              enableHiveSync, maxPendingCompactions, maxPendingClustering, continuousMode,
               minSyncIntervalSeconds, sparkMaster, commitOnErrors,
               deltaSyncSchedulingWeight, compactSchedulingWeight, deltaSyncSchedulingMinShare,
-              compactSchedulingMinShare, forceDisableCompaction, checkpoint,
+              compactSchedulingMinShare, forceDisableCompaction, forceDisableClustering, checkpoint,
```

Comment:
I see a config called compactSchedulingMinShare. Is there any necessity to create one for clustering?

Reply:
This config will be required only if we create a separate job pool for clustering and assign weights. As mentioned [here](https://github.com/apache/hudi/pull/3142#issuecomment-869829116) we are not creating a separate pool.